frontend/app/controllers/Joiner.scala

```diff
@@ -170,11 +183,25 @@ trait Joiner extends Controller with ActivityTracking {
       Future.sequence(optFuture.toSeq).map(_.headOption)
     }
 
+    val refererCookie = request.cookies.get("GU_MEM_REFERER").map(_.value.replace(Config.guardianUrl, ""))
```

Comment:
Just realised that the `val refererCookie` could be called `path` as that's what is being formed at this point?